sample/development/components/integration/test/integrationTests.brs

```diff
@@ -138,6 +138,105 @@ function TS_SDK_integration() as object
             return validator
         end function,
 
+        TC_SDK_getECID: function() as dynamic
```

Comment:
Can you add a test where getExperienceCloudId is called after setexperienceCloudID?